javascript/javascript3/week2/VisualPromise/main.js

```diff
@@ -0,0 +1,43 @@
+// This code will move the li to the position 100, 100. Calling moveElement will return a promise that resolves after the li element has been moved.
+const redBox = document.querySelector('ul.marks li:nth-child(1)');
+const blueBox = document.querySelector('ul.marks li:nth-child(2)');
+const greenBox = document.querySelector('ul.marks li:nth-child(3)');
+async function translateOneBtOne() {
+  redBox.style.left = 0;
+  redBox.style.right = 0;
+  blueBox.style.top = 0;
+  blueBox.style.left = 0;
+  greenBox.style.top = 0;
+  greenBox.style.left = 0;
+
```

Comment:
while styling don't use px because pixels is the static way. always try to use em,rem or %(these are the dynamic ways to define the size).

Reply:
@piepear  I think it is fine to use 0 without a unit, because 0 is 0 it does not mather what comes after it.

Reply:
@KavyaReddy732 I think that is not always bad to use pixels, for instance when you dealing with margin, padding and borders and many more cases when it would be better to them.

Reply:
I just meant that style should be assigned as a string in JS, not a number.  I didn't mean to start a CSS war here! 😄 😄

Reply:
@piepear if its a 0 and there is no unit, it is fine, no need to be a string. 
This is one way of learning from each other,  so this is not a CSS war, just a special operation 😀